sound/soc/sof/debug-dsp-ops.c

```diff
@@ -120,6 +120,38 @@ static ssize_t sof_dsp_ops_tester_dfs_read(struct file *file, char __user *buffe
 		default:
 			break;
 		}
+	} else if (!strcmp(dentry->d_name.name, "fw_state")) {
+		switch (sdev->fw_state) {
+		case SOF_FW_BOOT_NOT_STARTED:
+			string = "NOT STARTED\n";
+			break;
+		case SOF_DSPLESS_MODE:
+			string = "DSPLESS MODE\n";
+			break;
+		case SOF_FW_BOOT_PREPARE:
+			string = "PREPARE\n";
+			break;
+		case SOF_FW_BOOT_IN_PROGRESS:
+			string = "IN PROGRESS\n";
+			break;
+		case SOF_FW_BOOT_FAILED:
+			string = "FAILED\n";
+			break;
+		case SOF_FW_BOOT_READY_FAILED:
+			string = "READY FAILED\n";
+			break;
+		case SOF_FW_BOOT_READY_OK:
+			string = "READY OK\n";
+			break;
+		case SOF_FW_BOOT_COMPLETE:
+			string = "COMPLETE\n";
```

Comment:
I kind of like the numeric values as well (got used to them in the kernel log)...
Not sure if it makes sense to include it in the print (`6: BOOT_COMPLETE`)